docs/feb2020-workshop-agenda.md

```diff
@@ -0,0 +1,65 @@
+## Cylc Workshop
+### 10-14 Feb 2020, NIWA, Wellington, NZ
+
+#### Attendees:
+
+- Hilary Oliver, NIWA
+- Bruno Kinoshita, NIWA
+- David Sutherland, NIWA
+- Mel Hall, Met Office
+- Tim Pillinger, Met Office
+- Oliver Sanders, Met Office
+- David Matthews, Met Office
+- 
+
+### Workshop Goals
+- Primarily: review Cylc 8 progress to date and map out our roadmap to
+  completion
+- Secondarily: learn as much as we can about parts of the system mostly
+  worked on by others, to gain understanding and enable future work
+
+## Prelimary Agenda Topics
+
+(TBD: collate with others' suggestions and make a timetable)
+
+- architecture
```

Comment:
Not sure if this is the right section but I think we should include some discussion around how we support cli access via the hub (including for task communication purposes)

Reply:
I think you mean UI Server, not Hub, right?

Reply:
I assume you have to go via Hub to get to the UI server?